waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -364,3 +390,53 @@ proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
     await pm.connectToNodes(outsideBackoffPeers[0..<numPeersToConnect], WakuRelayCodec)
 
     await sleepAsync(ConnectivityLoopInterval)
+
+# Ensure we are always connected to the slotted service peers
+proc serviceConnectivityLoop*(pm: PeerManager) {.async.} =
```

Comment:
What would be the main advantage of maintaining our connections to service peers? Most service protocols are very opportunistic and may work fine with ad-hoc connections (e.g. when making a store query, filter query, etc.). In fact, we assume that client nodes making use of services may often have connectivity restrictions, so will for periods of time be unable to reach the service nodes. It is up to the application then to attempt connecting to service nodes (store, filter) to retrieve messages it may have missed during bad connectivity.

Certainly agree that we need to keep slots for service peers, though.

Reply:
main reasons are to i) shorten the time, since the connection is already made and ii) detect slot peers problems asap, and not wait until we need it. But I see your point.

Will remove it, but perhaps we will still have a similar behaviour? I mean, with keepAlive and one store request we will keep connected, until ofc the keepAlive fails or the connection is dropped.

Reply:
removed the serviceConnectivityLoop